meta-balena-thud/recipes-core/systemd/systemd/0003-Don-t-run-specific-services-in-container.patch.old

```diff
@@ -1,58 +0,0 @@
-From 514d33e874178d0ae7601b1363d2008bb78d1230 Mon Sep 17 00:00:00 2001
```

Comment:
Possible to squash this commit I guess?

Reply:
No, because this wrong push comes from sumo layer which was the base for creating this support. So I'd rather keep the history clean and not hide this in the thud layer creation based on the sumo one.

Reply:
ah. thought this came in the thud commit.